### PR TITLE
Update OWNERS

### PR DESCRIPTION
**Pull request by danpopSD**
would like to added as owner

Signed-off-by: danpopsd <dan.papandrea@sysdig.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

> /kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

> /area documentation

> /area videos

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by leodido** [comment]
As discussed privately with @danpopSD this PR is ok as long he's on the reviewers list.

As per guidelines, after various contributions, he can be in the approvers too.

File: OWNERS [hunk 1]
```diff
@@ -1,6 +1,7 @@
 approvers:
   - fntlnz
   - leodido
+  - danpopsd
```
Comment (leodido) [inline OWNERS:4]:
```suggestion
```

**Review by leodido — COMMENTED** [review]
LGTM as long you start as a reviewer (that's the process u frà you know :))

**Review by leogr — APPROVED** [review]
As per [this comment](https://github.com/falcosecurity/falco-website/pull/365#issuecomment-764505811), LGTM now!

**Pull request merged**